contents/handbook/growth/sales/how-we-work.md

```diff
@@ -122,7 +125,22 @@ The objective of the meeting is to hold each other to account, provide direct fe
   - Commission payments are made at the end of January, April, July, and October - at the end of each quarter, we'll monitor how many invoices actually get paid in the first two weeks of the next quarter. Fraser will send you an email that breaks down your commmission into the above 4 buckets and how you did. 
 - In your first 3 months you will not have a set quota but are expected to retain your existing book and have closed at least one Medium deal (either totally new or converting an existing customer to annual) - you'll be paid 100% OTE for this period.
 
-> Your quota and assigned customers are likely to change slightly from quarter to quarter. In any case, your quota will be amended appropriately (up or down) to account for any movement. We will also be flexible in making changes mid-quarter if it's obviously the sensible thing to do. If you inherit a new account, you have a 3 month grace period - if they churn in that initial period, they won't be counted against your quota. 
+> Your quota and assigned customers are likely to change slightly from quarter to quarter. In any case, your quota will be amended appropriately (up or down) to account for any movement. We will also be flexible in making changes mid-quarter if it's obviously the sensible thing to do. If you inherit a new account, you have a 3 month grace period - if they churn in that initial period, they won't be counted against your quota.
+
+## How contractual bonus works - CSMs
+
+CSMs are responsible for ensuring that a larger book of existing customers - both annual and monthly - continue to use PostHog successfully. They nurture customers and are product experts - this isn't a role of just going back and forth between customers and support engineers, or collecting feedback. 
+
+> This plan will _also_ almost certainly change as we scale up the size and complexity of our sales machine! As above, we will always ensure folks are treated fairly when we make changes.
+
+**Variables**
+
+- Your OTE comprises a 90/10 split between base and contractual bonus.
+- Bonus is paid based on logo retention above 90%, and is capped at 100%.
+  - For example, if you have 90% logo retention, you get 0% of bonus. For 95% retention, it's 50% bonus, and for 100% retention, it's 100% bonus. 
+- Bonuses are paid out quarterly, and in any case after an invoice is paid
+  - Bonus payments are made at the end of January, April, July, and October - at the end of each quarter, we'll monitor how many invoices actually get paid in the first two weeks of the next quarter. Fraser will send you an email that breaks down how you did.
+- We count a customer as having been retained in your book if they are still spending > $1,000/month with us, so if for example you have a customer that was spending $20k+ annually who then scales down to $10/year, they obviously won't count as retained even if they are technically still spending money!
```

Comment:
Disagree with this - part of the job might be right-sizing someone's spend so that they stay with us long term.

Reply:
What is the floor we should put here? It will be very easy for a CSM to game this by 'right sizing' someone to $100/mo, which I think just counts as churn. Maybe $1000/mo is too high.

Reply:
$500 ?

Reply:
done